### PR TITLE
Emagging a R&D console now unlocks it and prevents it from being locked as opposed to nothing

### PR DESCRIPTION
**Pull request by silicons**
I could make it corrupt the circuit so you'd have to swap out the circuit instead of screwdrivering it to fix it, y/n?

### CODE REVIEW

**Comment by Shadowflame909** [comment]
I'm glad it does something. I thought that's what it did before though. At least that's what it said it did?

**Comment by lordsaladin** [comment]
@kevinz000  add some syndie gear that you can make to the sci protolathe.

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -193,9 +193,10 @@ doesn't have toxins access.
 
 /obj/machinery/computer/rdconsole/emag_act(mob/user)
 	if(!(obj_flags & EMAGGED))
-		to_chat(user, "<span class='notice'>You disable the security protocols</span>")
+		to_chat(user, "<span class='notice'>You disable the security protocols [locked? "and unlock the console.":""].</span>")
```
Comment (SpaceManiac) [inline code/modules/research/rdconsole.dm]: This will have an extra space if the console is not locked.

File: code/modules/research/rdconsole.dm [hunk 2]
```diff
@@ -818,6 +819,9 @@ doesn't have toxins access.
 	if(ls["ui_mode"])
 		ui_mode = text2num(ls["ui_mode"])
 	if(ls["lock_console"])
+		if(obj_flags & EMAGGED)
+			to_chat(usr, "<span class='boldwarning'>Security protocal error: Unable to lock.</span>")
```
Comment (SpaceManiac) [inline code/modules/research/rdconsole.dm]: protocal

**Review by SpaceManiac — CHANGES_REQUESTED** [review]
Not sure if you want to do this at the same time or not but protolathes and circuit imprinters don't even have `emag_act` despite having "security protocols" mentioned in their UI.

**Review by SpaceManiac — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
this is a tame function and is fine, but please don't think the catch-all item needs to affect every machine :(

**Comment by AlexTheSysop42** [comment]
@lordsaladin scanning a traitor item already does that

**Pull request merged**